dist/index.html

```diff
@@ -14,6 +14,9 @@
     <link rel="stylesheet"
         href="//fonts.googleapis.com/css?family=Space+Mono" >
 
+     <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css?family=Open+Sans:300">     
```

Comment:
Don't double up on `link` tags here. Add the Open Sans reference to the existing `link` tag's `href` query string.